packages/grid/src/Grid.tsx

```diff
@@ -1682,43 +1662,44 @@ class Grid extends PureComponent<GridProps, GridState> {
     this.startDragTimer(event);
   }
 
-  handleDoubleClick(event: MouseEvent): void {
+  handleDoubleClick(event: React.MouseEvent): void {
     this.notifyMouseHandlers('onDoubleClick', event);
   }
 
-  handleMouseMove(event: MouseEvent): void {
+  handleMouseMove(event: React.MouseEvent): void {
     this.notifyMouseHandlers('onMove', event);
   }
 
-  handleMouseLeave(event: MouseEvent): void {
+  handleMouseLeave(event: React.MouseEvent): void {
     this.notifyMouseHandlers('onLeave', event, false);
     this.setState({ mouseX: null, mouseY: null });
   }
 
   handleMouseDrag(event: MouseEvent): void {
     this.setState({ isDragging: true });
-    this.notifyMouseHandlers('onDrag', event, true, true);
+    this.notifyMouseHandlers(
+      'onDrag',
+      (event as unknown) as React.MouseEvent,
```

Comment:
Can the event param just be a `React.MouseEvent`? Looks like you have that set for some other params here

Reply:
See my other comment about making a GridMouseEvent